meta-multimedia/recipes-multimedia/clight/clight_4.11.bb

```diff
@@ -0,0 +1,29 @@
+SUMMARY = "User daemon for automatic display management and ambient light adaptation"
+HOMEPAGE = "https://github.com/FedeDP/Clight"
+SECTION = "utils"
+
+LICENSE = "GPL-3.0-only"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
+
+SRCREV = "6a5ad96ce7643c433ea2f6bd597bc52dcc744120"
+SRC_URI = "git://github.com/FedeDP/${BPN};protocol=https;branch=master;tag=${PV} \
+"
+
+inherit cmake pkgconfig
+
+DEPENDS += "\
+    dbus \
+    gsl \
+    libconfig \
+    libmodule \
+    popt \
+    systemd \
+    udev \
+"
+
+RDEPENDS:${PN} = "clightd"
```

Comment:
since `clightd` depends on polkit distro features, this recipe should depend on that too. 

```
ERROR: Nothing RPROVIDES 'clightd' (but /mnt/jenkins/workspace/Yocto-world-clang/sources/meta-openembedded/meta-multimedia/recipes-multimedia/clight/clight_4.11.bb RDEPENDS on or otherwise requires it)
clightd was skipped: missing required distro feature 'polkit' (not in DISTRO_FEATURES)
```